tools/topology/m4/pipeline.m4

```diff
@@ -91,7 +91,7 @@ define(`PIPELINE_PCM_DAI_ADD',
 
 dnl PIPELINE_ADD(pipeline,
 dnl     pipe id, max channels, format,
-dnl     frames, deadline, priority, core)
+dnl     frames, deadline, priority, core, sched_comp)
```

Comment:
Isn't it already covered by PR #1075?

Reply:
yes, this is cleanup one, @ranj063 is closing #1075